src/server/controllers/users.controller.js

```diff
@@ -34,3 +12,15 @@ export function getAll(req, res, next) {
     }
   });
 }
+
+// Get all user list except logined user
+export function getMentorList(req, res, next) {
+  user.find({ 'email' : { $ne : 'session@yoda.com' }},  // if session_auth add, 'session' => req.session.email
```

Comment:
The userID I mean here is a numeric value, assigned to each user. That would be Yoda specific user ID which we issue, and not the FB/LI ids.